### PR TITLE
feat: Support FormPostHTMLTemplate config for fosite

### PR DESCRIPTION
**Pull request by wangleibeckham**
Support FormPostHTMLTemplate configuration during fosite init phase

## Related Issue or Design Document
https://github.com/ory/fosite/issues/646

## Checklist

- [x] I have read the [contributing guidelines](../blob/master/CONTRIBUTING.md)
      and signed the CLA.
- [x] I have referenced an issue containing the design document if my change
      introduces a new feature.
- [x] I have read the [security policy](../security/policy).
- [x] I confirm that this pull request does not address a security
      vulnerability. If this pull request addresses a security vulnerability, I
      confirm that I got green light (please contact
      [security@ory.sh](mailto:security@ory.sh)) from the maintainers to push
      the changes.
- [x] I have added tests that prove my fix is effective or that my feature
      works.
- [x] I have added necessary documentation within the code base (if
      appropriate).

### CODE REVIEW

**Comment by aeneasr** [comment]
Thank you for the PR! From the code I see that the new config value is not being used anywhere? We usually do not add code to Fosite that is not used by Fosite.

**Review by aeneasr — CHANGES_REQUESTED** [review]
Thank you for the PR! From the code I see that the new config value is not being used anywhere? We usually do not add code to Fosite that is not used by Fosite.

File: compose/compose.go [hunk 1]
```diff
@@ -74,6 +74,7 @@ func Compose(config *Config, storage interface{}, strategy interface{}, hasher f
 		ClientAuthenticationStrategy: config.GetClientAuthenticationStrategy(),
 		ResponseModeHandlerExtension: config.ResponseModeHandlerExtension,
 		MessageCatalog:               config.MessageCatalog,
+		FormPostHTMLTemplate:         config.FormPostHTMLTemplate,
```
Comment (mitar) [inline compose/compose.go:77]: I think it is being used here?

https://github.com/ory/fosite/blob/cf02af977681fd667b33f8e131891f6746d0b9da/fosite.go#L114

Comment (aeneasr) [reply compose/compose.go:77]: You're right, I thought that was also an addition, but it's not. Thank you, LGTM!

**Pull request merged**